region/crateria/west/Green Pirates Shaft.json

```diff
@@ -565,11 +565,12 @@
         {"or": [
           "h_canArtificialMorphIBJ",
           {"and": [
-            "SpringBall",
-            {"ammo": { "type": "PowerBomb", "count": 2}},
+            "h_canArtificialMorphSpringBall",
+            "h_canArtificialMorphPowerBomb",
+            "h_additionalBomb",
             {"or": [
               "HiJump",
-              {"ammo": { "type": "PowerBomb", "count": 1}},
+              "h_additionalBomb",
               "canJumpIntoIBJ"
             ]}
```

Comment:
Potentially a separate issue, I notice this PB doesn't help you jump high enough, but combining it  with canSpringBallBombJump would work.  The [4,10] version too.
The `canJumpIntoIBJ` seems hard at first but has a good visual cue with the side of the platform.